lib/puppet/face/node/clean.rb

```diff
@@ -53,12 +47,12 @@ def cleanup(node)
 
   # clean signed cert for +host+
   def clean_cert(node)
-    if Puppet::SSL::CertificateAuthority.ca?
-      Puppet::Face[:ca, :current].revoke(node)
-      Puppet::Face[:ca, :current].destroy(node)
-      Puppet.info _("%{node} certificates removed from ca") % { node: node }
-    else
-      Puppet.info _("Not managing %{node} certs as this host is not a CA") % { node: node }
+    begin
+      require 'puppetserver/ca/cli'
+      Puppetserver::Ca::Cli.run("clean", "--certname", node)
```

Comment:
This will go to STDOUT and STDERR by default. Is that something we do here? (Should we capture the output and print it using Puppet's logger? Should we wrap Puppet's logger and pass it into Cli? Does it matter? If this is already doing logging do we need the below message?

Reply:
Probably the most correct thing to do would be to capture the output and route it through puppet's logging. As is, I wanted to log something via puppet in case someone is ignoring STDOUT and STDERR in favor of wherever puppet's log are being directed to. I'll just update it to be more correct.

Reply:
So capturing the messages and feeding them to puppet's logger does an ugly thing:
```
Error: Error:
    Could not find files to clean for oops
```

And when I made a wrapper, I got THREE "Errors"!
```
Error: Error:
Error:     Could not find files to clean for yat
```

We could do some processing, but I really am curious how much it matters...

Reply:
By which I mean, maybe we should just let it go to standard out. Turns out `info` level logging is off by default anyway...